Expensify/App
# Fix saving reusable credentials

cc @tgolen 

### Details
I think infinite sessions was accidentally broken by https://github.com/Expensify/Expensify.cash/pull/1025

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/149153

### Tests
1. Log in
2. Verify you have the temporary login and password saved in storage like so
![2020-12-24_14-30-22](https://user-images.githubusercontent.com/32969087/103111769-9c352f00-45f4-11eb-85b4-107d479e1ee3.png)
3. Invalidate your `authToken` (wait two hours or follow similar technique explained [here](https://github.com/Expensify/Web-Expensify/pull/23787#issue-247086299))
4. Verify you are not logged out

### Tested On


### Screenshots
<Add any necessary screenshots for all platforms if your change added or updated UI>

#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
